src/runtimes/node/bundlers/nft/index.ts

```diff
@@ -113,6 +113,37 @@ const getTSModuleFormat = async (mainFile: string, repositoryRoot?: string): Pro
   return MODULE_FORMAT.COMMONJS
 }
 
+const getTypeScriptTransformer = async (runtimeAPIVersion: number, mainFile: string, repositoryRoot?: string) => {
+  const isTypeScript = tsExtensions.has(extname(mainFile))
+
+  if (!isTypeScript) {
+    return
+  }
+
+  const aliases = new Map<string, string>()
+  const rewrites = new Map<string, string>()
```

Comment:
why are we returning two empty maps in this case of v2?

Reply:
So that we can mutate them here: https://github.com/netlify/zip-it-and-ship-it/blob/fd3755c3c9b9d6cda44a05e804661fc806a8ed33/src/runtimes/node/bundlers/nft/index.ts#L195.